doc/rst/source/modules.rst

```diff
@@ -234,6 +233,7 @@ Core Modules
     - :doc:`grdconvert`
     - :doc:`grdcut`
     - :doc:`grdedit`
+    - :doc:`grdgdal`
     - :doc:`grdfft`
     - :doc:`grdfill`
     - :doc:`grdfilter`
```

Comment:
```suggestion
    - :doc:`grdfilter`
    - :doc:`grdgdal`
```

Reply:
thanks, I didn't know my brain told me **g** is before **f** 😄